app/models/concerns/pageflow/hosted_file.rb

```diff
@@ -82,10 +75,10 @@ def self.columns(t)
       t.integer(:attachment_on_filesystem_file_size, limit: 8)
       t.datetime(:attachment_on_filesystem_updated_at)
 
-      t.string(:attachment_on_s3_file_name)
-      t.string(:attachment_on_s3_content_type)
-      t.integer(:attachment_on_s3_file_size, limit: 8)
-      t.datetime(:attachment_on_s3_updated_at)
+      t.string(:attachment_file_name)
+      t.string(:attachment_content_type)
+      t.integer(:attachment_file_size, limit: 8)
+      t.datetime(:attachment_updated_at)
```

Comment:
Since the method is already deprecated and renaming the column here would be the sort of inconvenient breaking change the method was originally deprecated for, I suggest we remove it in 13.0 just like `keep_on_filesystem`.

Reply:
It's still referenced in the docs, though (`creating_file_types.md`). I'll remove it and update the doc accordingly